releases/app.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
-  name: vote-app
+  name: js-app
   namespace: arc-k8s-demo
 spec:
   releaseName: arc-k8s-demo
   chart:
-    git: https://github.com/Azure/arc-helm-demo
+    git: https://github.com/akashkeshari/pipelines-javascript-docker
     ref: master
-    path: charts/azure-vote
+    path: charts/azure-js
   values:
-    frontendServiceName: arc-k8s-demo-vote-front
\ No newline at end of file
+    frontendServiceName: arc-k8s-demo-js
```

Comment:
Let's make it azure-arc-sample or arc-k8s-demo. Drop the js

Reply:
done